coreax/kernel.py

```diff
@@ -348,6 +348,45 @@ def calculate_kernel_matrix_row_sum_mean(
         return self.calculate_kernel_matrix_row_sum(x, max_size) / (1.0 * x.shape[0])
 
 
+class LinearKernel(Kernel):
+    """Define a linear kernel."""
+
+    length_scale: float = 1.0
+    output_scale: float = 1.0
+
+    @override
+    def compute_elementwise(
```

Comment:
Remove the comma after each `y: Arraylike`, then when you re-run ruff format these should all fit on one line (see the base class for example).

Reply:
Whoops, fixed